src/refiners/training_utils/clock.py

```diff
@@ -22,7 +22,7 @@ def __init__(
         self,
         training_duration: TimeValue,
         gradient_accumulation: Step,
-        lr_scheduler_interval: TimeValue,
+        lr_scheduler_interval: Iteration | Epoch,
```

Comment:
Perhaps add an inline comment for Epoch (FIXME) - WDYT?

Reply:
Actually this attribute is useless, I removed it (and added the comment elsewhere).